code/game/objects/items/storage/belt.dm

```diff
@@ -738,7 +755,7 @@
 	new /obj/item/soap/nanotrasen(src)
 	new /obj/item/holosign_creator/janibarrier(src)
 	new /obj/item/melee/flyswatter(src)
-	new /obj/item/reagent_containers/cup/bucket(src)
+	new /obj/item/melee/baton/loaded(src)
```

Comment:
Is this intentional?

Reply:
Yes, the belt name is a misnomer, its only used by janitor ERT.

Due to Ruko's storage rebalancing, I've had to move a lot of items around so they actually fit when the outfit is equipped. In this case, batons are a large item that didnt fit anywhere else on the janitor ert, so its now in the belt.

Ive changed the name of the belt subtype so its easily known its for ERT. 56c5439